app/submitter/converter.py

```diff
@@ -128,3 +118,22 @@ def _build_metadata(metadata):
         downstream_metadata["display_address"] = metadata["display_address"]
 
     return downstream_metadata
+
+
+def get_optional_payload_properties(metadata, response_metadata):
+    payload = {}
+
+    if channel := metadata.get("channel"):
+        payload["channel"] = channel
+    if case_type := metadata.get("case_type"):
+        payload["case_type"] = case_type
+    if form_type := metadata.get("form_type"):
+        payload["form_type"] = form_type
+    if region_code := metadata.get("region_code"):
+        payload["region_code"] = region_code
+    if started_at := response_metadata.get("started_at"):
+        payload["started_at"] = started_at
+    if case_ref := metadata.get("case_ref"):
+        payload["case_ref"] = case_ref
```

Comment:
Maybe personal preference for DRY but this reduces the blockiness of the code for me
```suggestion
    for key in ["channel", "case_type", "form_type", "region_code", "started_at", "case_ref"]:
        if value := metadata.get(key):
            payload[key] = value
```